packages/dice-webapp/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,163 @@
+        <Hover
+          className={cn("rounded-full py-2 px-2", {
+            "mx-2": !isSidebarOpen,
+          })}
+          onClick={toggleSidebar}
+        >
+          <IoMenuOutline
+            className={cn("text-3xl text-black dark:text-white", {
+              "text-4xl": !isSidebarOpen,
+            })}
+          />
+        </Hover>
+        <div
+          className={cn("text-3xl text-black dark:text-white", {
+            "visibility: hidden": !isSidebarOpen,
+          })}
+        >
+          <FaDiceD20 />
+        </div>
+        <Image
+          className={cn("dark:invert", {
+            "visibility: hidden": !isSidebarOpen,
+          })}
+          src={LogoPath}
+          alt="DICE"
+        />
+      </section>
+      <main
+        className={cn("p-8 flex  flex-col items-start w-[full]  h-full", {
+          "w-[full] ": !isSidebarOpen,
+        })}
+      >
+        {orgSidebarData.map((d, i) => (
+          <>
+            <SidebarItem
+              activeIcon={d.activeIcon}
+              defaultIcon={d.defaultIcon}
+              isSidebarOpen={isSidebarOpen}
+              path={d.path}
+              title={d.title}
+            />
+          </>
+        ))}
+      </main>
+      <Button
+        color="danger"
+        variant="ghost"
+        onClick={() => {
+          signOut();
+        }}
+        className={cn(
+          "mx-4 my-4 p-3 text-base font-medium hover:font-bold w-[20%] h-[55px]",
+          { "visibility: hidden": !isSidebarOpen },
+        )}
+      >
+        <IoMdLogOut /> Logout
+      </Button>
+      <Button
+        isIconOnly
+        color="danger"
+        aria-label="Logout"
+        onClick={() => {
+          signOut();
+        }}
+        className={cn(
+          "text-white items-center text-3xl ml-12 my-4 w-[4%] h-[56px]",
+          {
+            "visibility: hidden": isSidebarOpen,
+          },
+        )}
+      >
+        <IoMdLogOut />
+      </Button>
+    </div>
+  );
+}
+
+type SidebarItemProps = {
+  title: string;
+  defaultIcon: React.ReactNode;
+  activeIcon: React.ReactNode;
+  path?: string;
+  isSidebarOpen: boolean;
+};
+
+function SidebarItem(props: SidebarItemProps) {
+  const pathname = usePathname();
+  return (
+    <Link href={props.path ?? "#"}>
+      <Hover
+        isActive={pathname === props.path}
+        className={cn("w-full flex items-center", {
+          "flex-col items-start gap-5 mx-2": !props.isSidebarOpen,
+        })}
+      >
+        <section
+          className={cn(
+            "text-3xl gap-4 flex items-center text-black dark:text-white",
+            {
+              "text-4xl": !props.isSidebarOpen,
+            },
+          )}
+        >
+          {props.path === pathname ? props.activeIcon : props.defaultIcon}
+          <p
+            className={cn("text-xl", {
+              "visibility: hidden": !props.isSidebarOpen,
+            })}
+          >
+            {" "}
```

Comment:
Why is an empty space used here?? Is this intentional?

Reply:
Removed unwanted spaces.